### ISSUE TITLE
"Virgin Islands (U.S.)" is parsed as "United States" instead of "Virgin Islands"

### ISSUE THREAD

**Issue by chriddyp** [vincentarelbundock/countrycode#133]
The string `Virgin Islands (U.S.)` is parsed as `United States` instead of `Virgin Islands`.

Originally reported in https://github.com/plotly/plotly.js/issues/1381

Plotly.js is using the regexes from this library (https://github.com/vincentarelbundock/countrycode/blob/master/data/countrycode_data.csv).

This string (`Virgin Islands (U.S.)`) was originally found in data from the World Bank.

Huge thanks for this library!

**Comment by vincentarelbundock** [comment]
Thanks. This was already added to the test suite (https://github.com/vincentarelbundock/countrycode/blob/master/tests/testthat/country_name_known_variations.json#L45), but I haven't found time to fix the regex yet.

Currently:

```
^(?=.*\bu\.?\s?s).*virgin|^(?=.*states).*virgin
united.?states\b(?!.*islands)|\bu\.?s\.?a\.?\b|^\s*u\.?s\.?\b(?!.*islands)
```

I suppose the second one should be fixed...

**Comment by cjyetman** [comment]
What version of countrycode are you using? This should be fixed in the most recent version (0.19) of countrycode on CRAN. Additionally, strings that have multiple matches should now return `<NA>`, so it should never return "United States". All in [commit ff530bb](https://github.com/vincentarelbundock/countrycode/commit/ff530bb9e38b87af986bf33b0e76432ec4f78179).

try running this code and paste the results...
```r
library(countrycode)
packageVersion('countrycode')
countrycode::countrycode('Virgin Islands (U.S.)', 'country.name', 'country.name')
```

this is what I get...
```r
> library(countrycode)
> packageVersion('countrycode')
[1] ‘0.19’
> countrycode::countrycode('Virgin Islands (U.S.)', 'country.name', 'country.name')
[1] "Virgin Islands, U.S."
```

**Comment by vincentarelbundock** [comment]
I like how @cjyetman offers more complete (and accurate) answers that the repo manager...
